README.md

```diff
@@ -76,6 +76,8 @@ curl -L https://raw.githubusercontent.com/canonical/cos-lite-bundle/main/overlay
 juju deploy cos-lite --channel=edge --trust --overlay ./offers-overlay.yaml
 ```
 
+To use COS-Lite with machine charms, see [cos-proxy](https://charmhub.io/cos-proxy) ([source](https://github.com/canonical/cos-proxy-operator)).
```

Comment:
It’s COS Lite, without dash

Reply:
Don’t you also yearn to link from here a tutorial or two @sed-i ?

Reply:
Not hyphenated anymore.
Is there a tutorial somewhere atm?